src/graph_gui.cpp

```diff
@@ -298,9 +311,96 @@ struct BaseGraphWindow : Window {
 		return max_width;
 	}
 
+	/**
+	 * Draw one graph plot and all de data points.
+	 * It also determines the minimal distance between the mouse click (this->click), and the current data point
+	 * @param id the ID of the plot about to be drawn
```

Comment:
We capitalize parameter descriptions and add a period at the end. (Throughout this PR)

```suggestion
	 * @param id The ID of the plot about to be drawn.
```